newsletter_laravel/app/Services/NewsletterService.php

```diff
@@ -0,0 +1,102 @@
+<?php
+namespace App\Services;
+
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Client;
+use Exception;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class NewsletterService {
+
+    private Client $client;
+
+    private SymfonyStyle $io;
+
+    public function __construct()
+    {
+        //Ne dela ce imam /services/v5 tukaj namesto v requestu?
+        $this->client = new Client(['base_uri' => 'https://www.pivotaltracker.com']);
+    }
+
+    /**
+     * Fetches all stories with the state set to accepted
+     *
+     * @param SymfonyStyle $io
+     * @return array|void|null
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function exportStories(SymfonyStyle $io)
+    {
+        $this->io = $io;
+        try {
+            $response = $this->client->request('GET', '/services/v5/projects/' . env('PIVOTAL_PROJECT_ID') . '/stories', [
```

Comment:
Extract this method call to new method like `loadStoriesForProject(int $projectId): array`